lib/db.js

```diff
@@ -84,7 +93,62 @@ class LimitDBRedis extends EventEmitter {
     });
   }
 
+  #setupPing() {
+    if (!this.pingConfig?.enabled) {
+      return;
+    }
+
+    this.failedPings = 0;
```

Comment:
Let's reset the ping always on 'ready.' Why only 'once' and not always? We can assume that 'ready' will re-install/setup ping?